fixtures/simple/README.md

```diff
@@ -0,0 +1,8 @@
+# Curated List of CFPs
+<!-- events -->
```

Comment:
@uchitsa maybe it's better to put here some other text, in order to test that your code really replaces this "another" text with the one generated from YAML?